packages/at_onboarding_cli/lib/src/onboard/at_onboarding_service_impl.dart

```diff
@@ -395,7 +436,7 @@ class AtOnboardingServiceImpl implements AtOnboardingService {
                     30)); // 30-second timeout should be enough even for slow networks
         connectionFlag = secureSocket.remoteAddress != null &&
             secureSocket.remotePort != null;
-      } on Exception catch (e) {
+      } on Exception catch (e, trace) {
```

Comment:
What happens if there is an Error rather than an Exception?

Reply:
I have handled error and logged